rest/data/impl.go

```diff
@@ -40,38 +40,7 @@ func (ctx *DBConnector) SetURL(url string)       { ctx.URL = url }
 func (ctx *DBConnector) GetPrefix() string       { return ctx.Prefix }
 func (ctx *DBConnector) SetPrefix(prefix string) { ctx.Prefix = prefix }
 
-type MockConnector struct {
-	URL    string
-	Prefix string
-
-	MockUserConnector
-	MockTaskConnector
-	MockContextConnector
-	MockDistroConnector
-	MockHostConnector
-	MockPodConnector
-	MockTestConnector
-	MockBuildConnector
-	MockVersionConnector
-	MockPatchConnector
-	MockPatchIntentConnector
-	MockProjectConnector
-	MockAdminConnector
-	MockStatusConnector
-	MockAliasConnector
-	MockRepoTrackerConnector
-	MockCLIUpdateConnector
-	MockGenerateConnector
-	MockSubscriptionConnector
-	MockNotificationConnector
-	MockCreateHostConnector
-	MockStatsConnector
-	MockTaskReliabilityConnector
-	MockCommitQueueConnector
-	MockSchedulerConnector
+type MockDBConnector struct {
+	DBConnector
+	MockDBConnectorImpl
```

Comment:
Can we call this MockGithubConnector to be clear about what kinds of things we need to mock?